README.md

```diff
@@ -61,9 +61,10 @@ _Features without a checkmark are still work in progress._
 ### For development use
 
 * Clone this repo into your plugins directory
-* Install the dependencies:  
+* Ensure you have Composer v2 and Node v16 installed
```

Comment:
Would it make more sense to explicitly mention `nvm use`?

Reply:
@tomjn I'm going to update it all soon anyway, the specific tool people use to manage node versions is up to them